.ci/scripts/create_simulator.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+xcrun simctl list
+xcrun simctl create Flutter-iPhone com.apple.CoreSimulator.SimDeviceType.iPhone-11 com.apple.CoreSimulator.SimRuntime.iOS-16-0 | xargs xcrun simctl boot
```

Comment:
Since we have a mini-script, I think it would be nice to pull out the device and OS into variables so each is on their own line instead of one massive line.